mangopay/__init__.py

```diff
@@ -1,7 +1,19 @@
+import os
+
+from .api import APIRequest  # noqa
+from .utils import memoize
+
 client_id = None
 apikey = None
-api_url = 'https://api.mangopay.com/v2.01/'
-api_sandbox_url = 'https://api.sandbox.mangopay.com/v2.01/'
+mangopay_url_env_var = os.getenv("MANGOPAY_URL")
+
+if mangopay_url_env_var is not None:
+    api_url = f'http://{mangopay_url_env_var}/v2.01/'
+    api_sandbox_url = f'http://{mangopay_url_env_var}/v2.01/'
```

Comment:
@kubaprzetakiewicz just a question though, do we need to add .sandbox to this url too or does your proxy work for both?